tests/Microsoft.Identity.Test.Unit/TelemetryTests/HttpTelemetryTests.cs

```diff
@@ -567,7 +567,9 @@ private async Task<(MockHttpMessageHandler MockHttpHandler, Guid Correlationid)>
 
             Assert.AreEqual(isCacheSerialized ? "1" : "0", telemetryCategories[2].Split(',')[0]); // is_cache_serialized
 
-            Assert.AreEqual(isLegacyCacheEnabled ? "1" : "0", telemetryCategories[2].Split(',')[1]); // is_legacy_cache_enabled
+            Assert.AreEqual(isLegacyCacheEnabled ? "1" : "0", telemetryCategories[2].Split(',')[1]); // is_legacy_cache_enabled
+
+            Assert.AreEqual("1", telemetryCategories[2].Split(',')[2]);
```

Comment:
Should also add tests/asserts for other telemetry token types.

Reply:
Updated a few tests to assert the token type set for auth scheme. This should cover the scenario that the correct value is set.